newIDE/app/src/ObjectEditor/Editors/SpriteEditor/CollisionMasksEditor/index.js

```diff
@@ -269,6 +274,18 @@ const CollisionMasksEditor = (props: Props) => {
               </Line>
             </React.Fragment>
           )}
+          {!!sprite &&
```

Comment:
I think we could move this into the `Hints/index.js` file? In an effort to centralise all this "extra knowledge" in a single place (no strong feeling on this though if this is complicated)